src/transcript/TranscriptResult.ts

```diff
@@ -2,11 +2,14 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import TranscriptAlternative from './TranscriptAlternative';
+import TranscriptLanguageWithScore from './TranscriptLanguageWithScore';
 
 export default class TranscriptResult {
   resultId: string;
   channelId?: string;
   isPartial: boolean;
+  languageCode?: string;
+  languageIdentifications: TranscriptLanguageWithScore[];
```

Comment:
Should `languageIdentifications` be optional? 

Also nit on the ordering of parameters, would prefer to add new ones to the bottom for consistency across all APIs and SDKs.

Reply:
Yes, similar to `languageCode`, `languageIdentifications` can be made optional as well. Will update in the next revision.